spec/components/support_interface/ucas_matches_statistics_component_spec.rb

```diff
@@ -42,48 +42,48 @@
   it 'renders number of candidates with dual application or multiple acceptances' do
     result = render_inline(described_class.new(ucas_matches))
 
-    expect(result.css('.app-card--blue')[0].text.strip).to include("7\n  candidates with dual application or multiple acceptances")
+    expect(result.css('.app-card--blue')[0].text.strip).to include("7\n    candidates with dual application or multiple acceptances")
```

Comment:
```suggestion
    expect(result.css('.app-card--blue')[0].text.squish).to include("7  candidates with dual application or multiple acceptances")
```
Using `squish` instead of `trim` enables us to have cleaner specs as what we are testing in this instance is the number of candidates and the correct text not not how the content is rendered.